applications/nfc/helpers/nfc_emv_parser.c

```diff
@@ -0,0 +1,49 @@
+#include "nfc_emv_parser.h"
+
+#include <file-worker.h>
+
+static bool
+    nfc_emv_parser_get_value(const char* file_path, string_t key, char delimiter, string_t value) {
+    bool found = false;
+    FileWorker* file_worker = file_worker_alloc(true);
+
+    if(file_worker_open(file_worker, file_path, FSAM_READ, FSOM_OPEN_EXISTING)) {
+        if(file_worker_get_value_from_key(file_worker, key, delimiter, value)) {
+            found = true;
+        }
+    }
+
+    file_worker_close(file_worker);
```

Comment:
You do not need to close the file before freeing the worker, this is done automatically

Reply:
Explicit better than implicit.